app/scripts/lib/constants.js

```diff
@@ -29,6 +29,8 @@ define(function (require, exports, module) {
 
     // Users that sign in to the content server directly
     CONTENT_SERVER_CONTEXT: 'web',
+
+    FX_SYNC_CONTEXT: 'fx_sync',
```

Comment:
The fx-sync broker is currently only instantiated directly whenever the user authenticates for sync and verifies in a 2nd browser. We could encode that into the name, in another branch I had a slightly different factoring where the FxSync broker stayed the same and I had a new broker, FxSyncVerificationBroker which was used when the user verifies in a 2nd browser. That factoring ended up being uglier than this factoring, the FxSyncVerificatonBroker took behaviors from FxSyncBroker, but nothing else.